builtin/credential/okta/path_config.go

```diff
@@ -80,18 +86,30 @@ func pathConfig(b *backend) *framework.Path {
 			},
 		},
 
-		Callbacks: map[logical.Operation]framework.OperationFunc{
-			logical.ReadOperation:   b.pathConfigRead,
-			logical.CreateOperation: b.pathConfigWrite,
-			logical.UpdateOperation: b.pathConfigWrite,
+		Operations: map[logical.Operation]framework.OperationHandler{
+			logical.ReadOperation: &framework.PathOperation{
+				Callback: b.pathConfigRead,
+				DisplayAttrs: &framework.DisplayAttributes{
+					OperationSuffix: "configuration",
+				},
+			},
+			logical.CreateOperation: &framework.PathOperation{
+				Callback: b.pathConfigWrite,
+				DisplayAttrs: &framework.DisplayAttributes{
+					OperationVerb: "configure",
```

Comment:
Will there be any limitations on what plugin authors can provide for `OperationVerb` or `OperationSuffix`? If so, is there a place for developers to discover this information?

Reply:
There are currently no limitations, just suggestions. In general the OperationID's will be constructed with `prefix` + `verb` + `suffix`. If not specified, these will be implied from the path / mount path. I'll see if we can add more guidelines to tutorials/docs.